src/util/file_util.cpp

```diff
@@ -61,11 +61,12 @@ std::string get_current_working_directory()
   std::string working_directory=wd;
   free(wd);
   #else
-  char buffer[4096];
+  wchar_t buffer[4096];
   DWORD retval=GetCurrentDirectory(4096, buffer);
   if(retval == 0)
     throw system_exceptiont("failed to get current directory of process");
-  std::string working_directory(buffer);
+  std::wstring unicode_buffer(buffer);
+  std::string working_directory(unicode_buffer.begin(), unicode_buffer.end());
```

Comment:
I'd suggest to use `std::string working_directory(narrow(buffer));` with an additional `#include "unicode.h"`.